packages/fxa-profile-server/test/api.js

```diff
@@ -875,8 +875,8 @@ describe('/avatar', function() {
 
   describe('upload', function() {
     it('should upload a new avatar', function() {
-      this.slow(2000);
-      this.timeout(3000);
+      this.slow(5000);
```

Comment:
Did the smaller instance size cause this to be slower?

Reply:
yes. it was right on the edge and failed sometimes on a `small`